0110/streams/index.html

```diff
@@ -1,275 +1,311 @@
-                    }
-
-                }
-            </pre>
-        </div>
-
-        <div class="code-example__snippet b-scala">
-            <pre class="brush: scala;">
-                import java.lang.Long
-                import java.util.Properties
-                import java.util.concurrent.TimeUnit
-
-                import org.apache.kafka.common.serialization._
-                import org.apache.kafka.streams._
-                import org.apache.kafka.streams.kstream.{KStream, KStreamBuilder, KTable}
-
-                import scala.collection.JavaConverters.asJavaIterableConverter
-
-                object WordCountApplication {
-
-                    def main(args: Array[String]) {
-                        val config: Properties = {
-                            val p = new Properties()
-                            p.put(StreamsConfig.APPLICATION_ID_CONFIG, "wordcount-application")
-                            p.put(StreamsConfig.BOOTSTRAP_SERVERS_CONFIG, "kafka-broker1:9092")
-                            p.put(StreamsConfig.DEFAULT_KEY_SERDE_CLASS_CONFIG, Serdes.String().getClass)
-                            p.put(StreamsConfig.DEFAULT_VALUE_SERDE_CLASS_CONFIG, Serdes.String().getClass)
-                            p
-                        }
-
-                        val builder: KStreamBuilder = new KStreamBuilder()
-                        val textLines: KStream[String, String] = builder.stream("TextLinesTopic")
-                        val wordCounts: KTable[String, Long] = textLines
-                            .flatMapValues(textLine => textLine.toLowerCase.split("\\W+").toIterable.asJava)
-                            .groupBy((_, word) => word)
-                            .count("Counts")
-                        wordCounts.to(Serdes.String(), Serdes.Long(), "WordsWithCountsTopic")
-
-                        val streams: KafkaStreams = new KafkaStreams(builder, config)
-                        streams.start()
-
-                        Runtime.getRuntime.addShutdownHook(new Thread(() => {
-                            streams.close(10, TimeUnit.SECONDS)
-                        }))
-                    }
-
-                }
-            </pre>
-        </div>
-    </div>
-    <h3 class="customer-title">See how Kafka Streams is being used</h3>
-    <div class="customer__cards">
-         <div class="customer__card">
-             <div class="customer__card__icon">
-            <img src="/{{version}}/images/icons/NYT.jpg" style="max-width:35%">
-          </div>  
-          <span class="customer__card__label">The New York Times uses Apache Kafka and the Kafka Streams API to store and distribute, in real-time, published content to the various applications and systems that make it available to the readers.</span>
-            <a href="https://www.confluent.io/blog/publishing-apache-kafka-new-york-times/">Learn More</a>
-             
+   <h1>Kafka Streams API</h1>
+   <ul class="streams-menu">
```

Comment:
When this element doesn't completely fit in view, we need to make it so users can scroll over  to see the rest of the menu. Take a look at .nav-scroller when the viewport is below 650px wide. Lets just do the same thing here.